app/components/cards/PostsList.jsx

```diff
@@ -124,11 +138,11 @@ class PostsList extends React.Component {
             (document.documentElement || document.body.parentNode || document.body).scrollTop;
         if (topPosition(el) + el.offsetHeight - scrollTop - window.innerHeight < 10) {
             const {loadMore, posts, category} = this.props;
-            if (loadMore && posts && posts.length > 0) loadMore(posts[posts.length - 1], category);
+            if (loadMore && posts && posts.size) loadMore(posts.last(), category);
         }
 
         // Detect if we're in mobile mode (renders larger preview imgs)
-        var mq = window.matchMedia('screen and (max-width: 39.9375em)');
+        const mq = window.matchMedia('screen and (max-width: 39.9375em)');
         if(mq.matches) {
             this.setState({thumbSize: 'mobile'})
```

Comment:
May need to check for this in shouldComponentUpdate -- this changes the size of loaded thumbs when window is resized between mobile/desktop view